examples/python/notebook/cube.ipynb

```diff
@@ -177,7 +177,6 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "\n",
```

Comment:
Wasn't this there on purpose?

Reply:
I have no idea what happened here - good catch